starter_transform_empdata_tosqlite.py

```diff
@@ -0,0 +1,420 @@
+#-*-coding: utf-8 -*-
+# Dev Mountain Tech Fest - Qualifiers Round
+# [python] xml/sqlite employee tranform data
+#
+# v1.0.0 09/10/65
+#   - Initial Release
+#
+# Author: Team MFEC
+
+
+# ================== Import library ==================
+import pprint   #pretty print
+import sqlite3  #sqlite
+import datetime #datetime
+import xml.etree.ElementTree as ET #xml reader
+from time import perf_counter #stopwatch
+from dateutil.relativedelta import relativedelta #timedelta
```

Comment:
This is a non standard library

Reply:
I agree with this mistake, maybe it came from installing some data sci lib for another project.

![Screen Shot 2565-10-13 at 00 13 35](https://user-images.githubusercontent.com/17945390/195527420-a7baacd3-87f0-4f9b-bf29-4a2e0ae2464f.png)

when we use this command and get no error so we decided to use it , sorry for the MISUNDERSTANDING.

  
  
ps. After I search for a while. The **PYTHON** standard date/datetime lib cannot calculate date difference with **YEARS** (unable to process 4year 1day rule) then it is impossible to do so

Reply:
Please give me the link

Reply:
> Please give me the link

@annibuliful   follow python's official documentation, the built-in lib datetime is not provided **YEARS** parameter with timedelta function
[https://docs.python.org/3/library/datetime.html#datetime.timedelta](https://docs.python.org/3/library/datetime.html#datetime.timedelta)

and One question please about date precision
The participants should be concern  about add extra 1day for every 4years pass or not ? for exactly date!
Cuz if some participant use "year" to minus year value directly that might be some deviation on "day"/"month" value

* test : before edit

Reply:
https://github.com/devmountaintechfest/hackathon-season2/pull/22/files#diff-d92aae6fcf09b8ae18a99689ed3006d0aca5a0e5216dc958dd0bd20ff33262c5R25

I checked from another team it seems work with standard lib

Reply:
> https://github.com/devmountaintechfest/hackathon-season2/pull/22/files#diff-d92aae6fcf09b8ae18a99689ed3006d0aca5a0e5216dc958dd0bd20ff33262c5R25
> 
> I checked from another team it seems work with standard lib

WOW that is great solution for calculate datetime! but there are 2 small bugs >>
1.  +2 years on `current_datetime.year + 2` , if current year is 2023 that will make **loop range to 2024** and get wrong +1 leap day cuz year2024 is Leap Year
2. current date and hired date are **not focus on february month** , if start day counting after the end of february month that might not necessary to add leap day

Reply:
1. Yes it should be <code>**_current_datetime.year + 1_**</code> instead of <code>**_current_datetime.year + 2_**</code> it was my mistake during the coding period.
2. You stated that it might not be necessary to add leap day which means you know that this can be solved by the policy.